### DISCUSSION TITLE
How to create round corner in LineDrawable?

### DISCUSSION THREAD

**Discussion by alich65** [mapsforge/vtm#899] [Q&A] [answered]
I have create a LineDrawable but corners are very sharp. Can I round those?
![Capture](https://user-images.githubusercontent.com/40024574/148960298-9103aa7e-5cdd-47fb-860d-31d968c96605.PNG)

This is my code:
`      
 Style.Builder sb = Style.builder()
.strokeWidth(5f) 
.strokeColor(Color.parseColor("#5069F3"));
        Style style = sb.build();
        LineDrawable lineDrawable = new LineDrawable(points, style);
`

**Comment by devemux86** [comment, marked as answer]
Round join is not available.
